src/devices/LiquidLevel/LiquidLevelSwitch.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.LiquidLevel
+{
+    /// <summary>
+    /// Supports any single pin output digital liquid level switch which is configured
+    /// </summary>
+    public class LiquidLevelSwitch : IDisposable
+    {
+        private readonly int _dataPin;
+        private readonly PinValue _liquidPresentPinState;
+        private readonly bool _shouldDispose;
+
+        private GpioController _controller;
+
+        /// <summary>Creates a new instance of the LiquidLevelSwitch.</summary>
+        /// <param name="dataPin">The data pin</param>
+        /// <param name="liquidPresentPinState">Whether the pin value used to detect liquid is low or high</param>
```

Comment:
```suggestion
        /// <param name="liquidPresentPinState">Data pin state representing liquid being present</param>
```

Reply:
Changes made, however 2 of the checks failed, looks like an issue with the builds pulling some .net 5 preview nuget packages.